rustup/build.go

```diff
@@ -120,7 +120,7 @@ func AdditionalTarget(cr libpak.ConfigurationResolver, stack string) string {
 	}
 
 	libc := "gnu"
-	if stack == libpak.TinyStackID {
+	if libpak.IsTinyStack(stack) || libpak.IsStaticStack(stack) {
```

Comment:
@dmikusa do you think it's worth extending this to include the Jammy Tiny stack? Given that the bionic stack is going EOL in about 6 weeks?

I could see the argument for making that a separate issue, but it seems tightly related to this change.

Reply:
These methods in libpak will match both. I did it this way so it leaves room for adding/removing stacks in the future without changing all the buildpacks.

Reply:
Ah, I didn't see that they were in the PR that's not yet merged: https://github.com/paketo-buildpacks/libpak/pull/229/files

Makes total sense.